src/ui/states/state-error.svelte

```diff
@@ -36,6 +36,8 @@
         return lastError.message;
       case PurchaseFlowErrorCode.MissingEmailError:
         return "Email is required to complete the purchase.";
+      case PurchaseFlowErrorCode.AlreadySubscribedError:
+        return "This user is already subscribed to this product."
```

Comment:
I would change this to "You are already subscribed to this product", since this is what the customer sees